### PR TITLE
Removed the code to import scenarios from connector

### PR DESCRIPTION
**Pull request by cs-swati-jinde**
Removed the code to import scenarios from connector

### CODE REVIEW

File: fortisoar-soc-simulator/info.json [hunk 1]
```diff
@@ -9,16 +9,6 @@
     "category": "information",
     "icon_medium_name": "fortisoar_medium.png",
     "configuration": {
-        "fields": [
```
Comment (cs-tushar-kanade) [inline fortisoar-soc-simulator/info.json:12]: Syntax error here , Fields [  ...

Comment (cs-tushar-kanade) [reply fortisoar-soc-simulator/info.json:12]: check RAW file , we see ] in file

File: fortisoar-soc-simulator/utils.py [hunk 2]
```diff
@@ -11,48 +11,6 @@
...
-                logger.debug("Deleting Configuration {}".format(scenario.title()))
-                id = item['@id'].split('/')[-1]
-                url = "/api/3/delete/import_jobs"
-                body = {"ids":[id]}
-                make_request(url,"DELETE",body=body)
-                logger.debug("Deleted Configuration {}".format(scenario.title()))
-
-    scenario_path=os.path.join(os.path.dirname(__file__), "scenarios/"+scenario+"/scenario_configuration.json")
-    if(os.path.exists(scenario_path)):
-        set_config_job(scenario_path,scenario_title=scenario)
```
Comment (cs-tushar-kanade) [inline fortisoar-soc-simulator/utils.py:52]: apart from import_records there are supporting functions , please remove them as well .e.g set_config_job , get_config_job

**Review by cs-tushar-kanade — CHANGES_REQUESTED** [review]
Please do changes are per comments

File: fortisoar-soc-simulator/info.json (hunk 1 above)
Comment (cs-swati-jinde) [reply fortisoar-soc-simulator/info.json:12]: Resolved, Please review

File: fortisoar-soc-simulator/utils.py (hunk 2 above)
Comment (cs-swati-jinde) [reply fortisoar-soc-simulator/utils.py:52]: Resolved, removed get_config_job(),set_config_job,_upload_file,_create_import_job

**Comment by cs-swati-jinde** [comment]
Following changes Done in Soc Simulator 2.0.0
1.Info.json - updated version, Removed the 'import scenario' checkbox option
2.Deleted the scenario folder with each scenario records

**Review by cs-tushar-kanade — APPROVED** [review]
Looks good

**Comment by cs-tushar-kanade** [comment]
DO NOT MERGE YET , WE DO NOT WANT TO RELEASE CONNECTOR

File: fortisoar-soc-simulator/utils.py [hunk 3]
```diff
@@ -11,83 +11,6 @@
...
-def _create_import_job(file_iri):
-    # payload = {"file" : {"@id":file_iri},"status":"Draft","options":{}}
-    payload = {"file": file_iri, "status": "Draft", "options": {}}
-    url = "/api/3/import_jobs"
-    response = make_request(url, 'POST', body=payload)
-    response = response.json()
-    job_id = response.get('@id').split("/")[-1]
-    return job_id
 
 def load_threat():
```
Comment (cs-suyog-jinturkar) [inline fortisoar-soc-simulator/utils.py:15]: What is this function do?

**Pull request merged**